### PR TITLE
Creates a Common Error String Resource File

### PR DESCRIPTION
**Pull request by asitava1998**
Fixes issue #569 

Changes: 
* Implements same set of strings across all experiment files
* Creates a common file for the same

Screenshots for the change: 
None

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
Hi @asitava1998 ! It's a good fix! What if we use a static class for strings? :)

**Comment by asitava1998** [comment]
@CloudyPadmal  We can do that too. Should I declare it as static then?

**Comment by CloudyPadmal** [comment]
Hi @asitava1998 ! If it's not much of a trouble! :)

File: app/src/main/java/org/fossasia/pslab/experimentsetup/ExperimentErrorStrings.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.fossasia.pslab.experimentsetup;
+
+/**
+ * Created by asitava on 27/8/17.
+ */
+
+public class ExperimentErrorStrings {
+    String ERROR_MESSAGE = "Invalid Value";
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/experimentsetup/ExperimentErrorStrings.java]: You can create a `static` class and define all `strings` as `static`. Then you won't need to create `ExperimentErrorStrings` object. Simply refer to `ExperimentErrorStrings.anystring`

**Comment by asitava1998** [comment]
@CloudyPadmal @viveksb007 I have made the changes. Declared the strings static.

**Pull request merged**